### ISSUE TITLE
Update data-raw/node_config.csv file

### ISSUE THREAD

**Issue by shahmanash** [AtlasOfLivingAustralia/galah-R#210]
Please update the [node_config.csv](https://github.com/AtlasOfLivingAustralia/galah-R/blob/main/data-raw/node_config.csv) file to include the Swedish namematching service.

```
Sweden,name-matching,names_lookup,https://namematching.biodiversitydata.se/api/getByTaxonID,search_identifiers,TRUE
Sweden,name-matching,names_search_multiple,https://namematching.biodiversitydata.se/api/searchByClassification,search_taxa,TRUE
Sweden,name-matching,names_search_single,https://namematching.biodiversitydata.se/api/search?q={name},search_taxa,TRUE
```
Please note the lucene search index for the service is generated using the GBIF backbone.

**Comment by mjwestgate** [comment]
Thanks for sending this information! We've updated `galah` as per your request, these changes will be shipped with version 2.0 shortly.

**Comment by mjwestgate** [comment]
The above were added in version 2.0, however the following need checking:

Shouldn't the line
`Sweden,data/media,https://images.biodiversitydata.se/ws/getImageInfoForIdList,FALSE`
be
`Sweden,metadata/media,https://images.biodiversitydata.se/ws/getImageInfoForIdList,TRUE`

Similarly "ws" is missing in line
`Sweden,metadata/licences,https://images.biodiversitydata.se/licence,TRUE`
which then should be
`Sweden,metadata/licences,https://images.biodiversitydata.se/ws/licence,TRUE`

The following 3 lines could be added
```
Sweden,files/media,https://images.biodiversitydata.se/image/{id}/original,TRUE
Sweden,metadata/lists,https://lists.biodiversitydata.se/ws/speciesList,TRUE
Sweden,metadata/lists-unnest,[https://lists.biodiversitydata.se/ws/speciesListItems/{list_id},TRUE
```
